services/ui-src/src/components/modals/AddEditProgramModal.test.tsx

```diff
@@ -22,17 +22,13 @@ const mockedReportContext = {
   fetchReportsByState: mockFetchReportsByState,
 };
 
-const mockNewReportData = {
-  reportType: "mock-type",
-  formTemplate: mockReportJson,
-};
-
 const modalComponent = (
   <ReportContext.Provider value={mockedReportContext}>
     <AddEditProgramModal
       activeState="AB"
       selectedReport={undefined}
-      newReportData={mockNewReportData}
+      reportType={"mock-type"}
```

Comment:
Nit: Braces not necessary here.
```suggestion
      reportType="mock-type"
```